roll/__init__.py

```diff
@@ -301,33 +310,138 @@ def cookies(self):
         return self._cookies
 
 
-class Protocol(asyncio.Protocol):
+class WSProtocol(WebSocketCommonProtocol):
+
+    needs_upgrade = True
+    allowed_methods = {'GET'}
+    timeout = 5
```

Comment:
Here and below, uppercase if constants? More consistent with other declarations.

Reply:
+1 for uppercase
Plus: what about reading the value from the env var, with a default?
Something like:

    TIMEOUT = os.environ.get('ROLL_WEBSOCKET_TIMEOUT', 5)

As general principle, I like the idea of having the first configuration level (setting up a configuration value) with env var, and the second level (changing the behaviour) from inheritance.